### DISCUSSION TITLE
QEMU Error When Using Windows VM

### DISCUSSION THREAD

**Discussion by CocoTech1861** [utmapp/UTM#5847] [UTM Help (macOS)]
When I try to launch my Windows 11 VM, I get [this error message](https://www.dropbox.com/scl/fi/qx5y09bjesfgnjl0sq7mq/UTM_Error.png?rlkey=acud1qz67rlus5yh6gvgly4x7&dl=0). I need this VM as it runs the lighting software I use, so this is kinda urgent. Please help.
-----
**Configuration**  
* UTM Version: **Version 4.4.4 (92)**
* macOS Version: **Version 14.0**
* Mac Chip **Apple M1 Pro**

**Debug log**  
https://www.dropbox.com/scl/fi/opcpegj28r2rtnp7ojfgy/debug.log?rlkey=upx5jsvxbvmz2du19f5exw83f&dl=0

**Upload VM**  
https://www.dropbox.com/scl/fi/ntgfi2mj258y866ooyfdo/config.plist?rlkey=bkepxe2wh2kl82rgot2epwmqo&dl=0

**Comment by osy** [comment]
Eject the ISO from the virtual disk drive.

**Reply by CocoTech1861** [reply]
@osy https://www.dropbox.com/scl/fi/nklzg6vigi9bd9oe46rwb/Screenshot-2023-10-30-at-6.45.40-PM.png?rlkey=hiwkpz9xrgrx9pk21tdxmlhqd&dl=0 Now I get this
